e2e/integration_tests/setup_test.ts

```diff
@@ -44,6 +45,17 @@ registerTestEnv({
 
 registerTestEnv({name: 'cpu', backendName: 'cpu', isDataSync: true});
 
+// TODO: Support test windows on WebGPU. Bug:
+// https://github.com/tensorflow/tfjs/issues/7616.
+if (navigator.platform.toUpperCase().indexOf('MAC') >= 0) {
```

Comment:
We should wait for Windows CI land and remove this.

Reply:
Can we make Windows CI workable?

Reply:
With below config on windows/browerstack, the gpu adapter is null.
```
const chromeWebgpuFlags = [
  '--enable-unsafe-webgpu',  // Can be removed after WebGPU release
  '--use-webgpu-adapter=swiftshader',
  // https://github.com/tensorflow/tfjs/issues/7631
  '--disable-vulkan-fallback-to-gl-for-testing',
];
```

Reply:
Let's merge this first and get Windows CI working after. I'm working with BrowserStack to debug it.